src/LupaClient.php

```diff
@@ -179,7 +180,7 @@ public function send(
                 $response = $this->getHttpClient()->send($request);
 
                 return JsonUtils::jsonDecode($response->getBody()->getContents(), true);
-            } catch (ConnectException $e) {
+            } catch (ConnectException|ServerException $e) {
```

Comment:
Union Types support starts at PHP 8.0 , this package supporting 7.2 PHP

Reply:
Catcham biski anksciau atsirado, nuo 7.1

As of PHP 7.1.0, a [catch](https://www.php.net/manual/en/language.exceptions.php#language.exceptions.catch) block may specify multiple exceptions using the pipe (|) character. This is useful for when different exceptions from different class hierarchies are handled the same.

https://www.php.net/manual/en/language.exceptions.php
https://wiki.php.net/rfc/multiple-catch